pkg/handlers/login.go

```diff
@@ -110,7 +110,29 @@ func (h *Handler) Login(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	loginResponse.Token = fmt.Sprintf("Bearer %s", signedJWT)
+	responseToken := fmt.Sprintf("Bearer %s", signedJWT)
+
+	expire := time.Now().Add(24 * time.Hour)
+	tokenCookie := http.Cookie{
+		Name:     "signed-token",
+		Value:    responseToken,
+		Expires:  expire,
+		Path:     "/",
+		HttpOnly: true,
+	}
+
+	originURL, err := url.Parse(r.Header.Get("Origin"))
```

Comment:
this seems like a similar scenario as [above](https://github.com/replicatedhq/kots/pull/3810/files#r1169465032).  we need to trust the `Origin` header to know if they are using https since it can vary from one enterprise instance to another?

Reply:
Yes, since the domain can change from customer to customer we have to use the value from the `origin` header to determine if the cookie needs to be set to `secure`.